setup.py

```diff
@@ -18,7 +18,7 @@
     author="Yoshiki Vazquez-Baeza",
     author_email="yoshiki@ucsd.edu",
     description="Display ordination plots",
-    license="bsd",
+    license='BSD-3-Clause',
```

Comment:
Sounds good.

On (Jul-19-16|16:38), Jorden Kreps wrote:
> > @@ -18,7 +18,7 @@
> >      author="Yoshiki Vazquez-Baeza",
> >      author_email="yoshiki@ucsd.edu",
> >      description="Display ordination plots",
> > -    license="bsd",
> > +    license='BSD-3-Clause',
> 
> @ElDeveloper 
> >  why are we changing this anyways then?
> 
> There was a huge mixture of license holders and version strings across all the repos in the qiime2 organization, so just went through and made them all consistent both on the setup.py license and in the license files.
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/qiime2/q2-emperor/pull/8/files/d569ddf053ba9da3e75066a080eafc6e1178fd3f#r71443646